grunttasks/l10n-extract.js

```diff
@@ -43,12 +43,7 @@ module.exports = function (grunt) {
 
     walker.on('end', function () {
       var jsWalker = extract({
-        'input-dir': pkgroot,
-        /* node_modules and test should not contain any strings
-         * Gruntfile causes an error and should contain no strings
-         * bin/server.js extracts "/", so it is excluded.
-         */
-        exclude: /(node_modules|test|Gruntfile|grunttasks|bin|scripts|\.git|config)/,
```

Comment:
Losing these excludes was another nice side-effect. None of these things pollute the same space as the mailer source any more.